message-hub/README.md

```diff
@@ -0,0 +1,174 @@
+# Using IBM Message Hub and Apache Kafka to copy documents from Cloudant to Db2 Warehouse on Cloud
+
+This tutorial will use Apache Kafka Connect to publish messages from a Cloudant
+database to an IBM Cloud Message Hub topic. The messages will then be consumed by
+Apache Kafka Connect's JDBC connector to write to a Db2 Warehouse on Cloud table.
+
+Note that this example uses the Kafka Connect standalone worker configuration.
+
+**N.B. This is an example only intended as a _starting point_ for using IBM Message Hub and Apache Kafka Connect to extract data from Cloudant and insert into Db2 Warehouse on Cloud. It is _not_ intended as a drop-in replacement for the deprecated Cloudant warehouse integration. In particular this example does not create or alter Db2 tables or handle document deletion.**
+
+# Setting up IBM Cloud services
+
+## Cloudant
+
+1. [Create an IBM Cloudant](../cloudant/create.md) instance.
+1. [Replicate the tutorial data](../cloudant/replicate.md).
+
+## Message Hub
+
+1. Create an [IBM Message Hub](https://console.bluemix.net/catalog/services/message-hub) service instance.
+![Message Hub in catalog](mh_catalog.png)
+![Create Message Hub instance](mh_create.jpg)
+1. Create Service Credentials for IBM Message Hub.
+![Create Message Hub service credentials](mh_creds.jpg)
+1. Use the Message Hub `Manage` interface to create a topic with a single partition, e.g. `exampleTopic`.
+![Create Message Hub topic](mh_topic.jpg)
+
```

Comment:
If you remove this space the "alternative" piece below will be indented to the same level as the rest of step 3.

Reply:
Done in 4a6a442d99210207d0321482c437bd43e9b1a687